plugins/woocommerce-blocks/assets/js/blocks/product-filters/block.json

```diff
@@ -44,7 +44,9 @@
 	},
 	"textdomain": "woocommerce",
 	"usesContext": [ "postId" ],
-	"providesContext": {},
+	"providesContext": {
+		"woocommerce/product-filters-overlay-navigation/overlay": "overlay"
```

Comment:
I'm following the recommendation from [the docs](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-context/). 

> Include a namespace
As seen in the above example, it is recommended that you include a namespace as part of the name of the context key so as to avoid potential conflicts with other plugins or default context values provided by WordPress. The context namespace should be specific to your plugin, and in most cases can be the same as used in the name of the block itself.

Reply:
Oh, I wasn't aware of that, thanks. TIL.